### DISCUSSION TITLE
Как уменьшить сталкера?

### DISCUSSION THREAD

**Discussion by Karpai** [PavelBlend/blender-xray#630] [Q&A] [answered]
Я хочу, чтобы в игре был сталкер примерно 80% от стандартного размера.
Уменьшив скелет, нпс стал меньше и рэгдолы выглядят хорошо, однако со многими анимациями, у них плечи разлетаются в разные стороны, шея вытягивается и иногда сталкер становится в полный рост.
Все анимации в Blender проигрываются правильно. Как их адаптировать, если я не вижу проблемы в программе?

Если нпс можно уменьшить через скрипты\конфиги не поломав физику, это тоже поможет.
Спасибо за этот чудесный плагин.

**Comment by PavelBlend** [comment, marked as answer]
@Karpai нужно поочерёдно импортировать все omf анимации, которые использует модель и заново их экспортировать (возможно с новыми именами или в другую папку, чтобы старые модели не испортить). Потом эти полученные файлы нужно прописать объекту в motion references. Во время экспорта omf или skls, аддон учитывает масштаб объекта.

**Reply by Karpai** [reply]
Надо было ещё перед этим некий bone group создать.
Спасибо. Неожидал, что это так просто.
